Program.cs

```diff
@@ -154,6 +154,20 @@ private static async Task<IReadOnlyList<EnumMemberDeclarationSyntax>> GetErrorCo
                 codes.Add(code, href);
             }
 
+            foreach (var item in root!.Items.Where(e => e.Items == null && !string.IsNullOrWhiteSpace(e.DisplayName)))
+            {
+                var displayName = item.DisplayName.Replace(", ", ",");
+                var subCodes = displayName.Split(',');
```

Comment:
Nit: you could use `Regex.Split(item.DisplayName, @"\s*,\s*")` to account for the optional whitespace.

Reply:
I like it, thanks for the suggestion!